components/script/script_thread.rs

```diff
@@ -1659,20 +1644,15 @@ impl ScriptThread {
                                  incomplete.window_size);
         let frame_element = frame_element.r().map(Castable::upcast);
 
-        enum ContextToRemove {
-            Root,
-            Child(PipelineId),
-            None,
-        }
         struct AutoContextRemover<'a> {
```

Comment:
We can remove this entirely; there are no early returns from this method any more.